ui/app/AppLayouts/Communities/popups/CreateCommunityPopup.qml

```diff
@@ -14,6 +14,7 @@ import StatusQ.Components 0.1
 import StatusQ.Controls 0.1
 import StatusQ.Controls.Validators 0.1
 import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
```

Comment:
`import QtQuick.Dialogs 1.3` above should be removed

Reply:
I couldn't remove that bc there is `MessageDialog` that was refactored in a separate pr. Now I think it's already in master, I'll rebase and review!

Reply:
Done!